src/Tribe/Attendee_Registration/View.php

```diff
@@ -3,19 +3,7 @@
 /**
  * Class Tribe__Tickets__Attendee_Registration__View
  */
-class Tribe__Tickets__Attendee_Registration__View extends Tribe__Template {
-	/**
-	 * Tribe__Tickets__Attendee_Registration__View constructor.
-	 *
-	 * @since 4.9
-	 */
-	public function __construct() {
-		$this->set_template_origin( tribe( 'tickets.main' ) );
-		$this->set_template_folder( 'src/views' );
-		$this->set_template_context_extract( true );
-		$this->set_template_folder_lookup( true );
-	}
-
+class Tribe__Tickets__Attendee_Registration__View extends Tribe__Tickets__Editor__Template {
```

Comment:
:no_entry_sign: Class name is not valid; consider Tribe_Tickets_Attendee_Registration_View instead

<sub>hash: bdad6d8a6dd00d4263ccc8dd59a428ec</sub>